src/logger/Logger.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+
+class Logger:
+    def __init__(self, debug=False, logfile='log.txt'):
+        self.debug = debug
+        self.logfile = logfile
+        self.f = open(logfile,'a+')
+
+    def __open(self):
+        self.f = open(self.logfile,'a+')
+
+    def __close(self):
+        self.f.close()
+
+    def log(self, message, severity='INFO'):
+        self.__open()
+        timestamp = datetime.datetime.now()
+
+        if severity == 'ERROR':
+            self.f.write('[ERROR]  ' + str(timestamp) + '  ' + message + '\r\n')
```

Comment:
I just tried it on Ubuntu 16, it prints two newlines. It looks like Python converts \r to \n

Reply:
Wait, no, \r prints nothing.

Reply:
I think it might be better to just use \n as our default line ending, since our application runs on linux anyways. Also plenty of Windows software knows how to convert \n to \r\n.

Reply:
\r shouldn't have an impact on Linux while being compatible with Windows but doing some reading Python apparently converts \n to the correct line ending for the platform so it isn't necessary and I'll remove it.